packages/components/ng-ovh-doc-url/src/provider.js

```diff
@@ -0,0 +1,27 @@
+export default function () {
+  const isValidLocaleFormat = new RegExp('^[A-Za-z]{2}_[A-Za-z]{2}');
```

Comment:
Looks like a job for a `LanguageService` so we don't have language logic spread everywhere in the codebase

Reply:
This PR target only the import of `@ovh-ux/ng-ovh-doc-url`. 
If we want to address this issue/enhancement, we should open an issue.

Reply:
Maybe we can open an Issue to address your comments @FredericEspiau.

Let me know what do you think. Thanks!